Makefile

```diff
@@ -228,7 +226,6 @@ docker-build-trivy-scanner: ## Build docker image for trivy-scanner image.
 	docker buildx build \
 		$(_CACHE_FROM) $(_CACHE_TO) \
 		$(_ATTESTATIONS) \
-		--build-arg TRIVY_BINARY_IMG="$(TRIVY_BINARY_IMG)" \
```

Comment:
Why are we removing this?

Reply:
Reverted this change. I was a bit confused since we define this in both Makefile and Dockerfile, but I see that Dockerfile gets overwritten by this (added a comment in Dockerfile). It would be good to have a single source of truth in the future.